bin/phpstan

```diff
@@ -24,6 +25,7 @@ use Symfony\Component\Console\Helper\ProgressBar;
 
 	$devOrPharLoader = require_once __DIR__ . '/../vendor/autoload.php';
 	require_once __DIR__ . '/../preload.php';
+	$vendorDirectory = ComposerHelper::getVendorDir(getcwd());
```

Comment:
This can return null which means it's gonna get concatenated lower leading to double `//` in the path...